### DISCUSSION TITLE
طراحی پوستر فراخوان

### DISCUSSION THREAD

**Discussion by FB-888222** [WeAreMahsaAmini/Core#316] [Ideas]
سلام دوستان
حامد اسماعیلیون و تیمش برای فراخوان تظاهرات مردمی برای هرشهر یک پوستر طراحی کردند و تاریخ و ساعت و مکان تجمعات رو نوشتن که برای روز شنبه ساعت ۱۲:۳۰ هست.
الان توی خیلی از شهر های ایران مردم بیرون میرن ولی چون فراخوان وجود نداره و محله محور نیست اعتراضات خیلی نتیجه نمیده.
پیشنهاد من این بود که دوستانی که میتونن مشابه پوستر های اقای اسماعیلیون برای شهر های ایران طراحی بشه و هم چنین نام دو یا سه محله از شهر های بزرگ نوشته بشه که مردم بدونن محل تقریبی کجاست.
ممنون از پروژه و کمک هاتون
#مهسا_امینی

**Comment by ghost** [comment]
#no_to_sedition
#نه_به_آشوب
#نه_به_اغتشاش
#صلح
#مرگ_بر_آشوبگر
